sql/src/test/java/io/crate/execution/dml/upsert/SourceFromCellsTest.java

```diff
@@ -337,7 +337,7 @@ public void test_nested_partition_column_is_not_included_in_the_source() throws
         Map<String, Object> obj = new HashMap<>();
         obj.put("x", 10);
         obj.put("p", 1);
-        var source = sourceGen.generateSourceAndCheckConstraints(new Object[] { obj });
+        var source = sourceGen.generateSourceAndCheckConstraintsAsBytesReference(new Object[] { obj });
```

Comment:
Minor: The tests would like much nicer on Map<String, Object> as a result type. If we go down this path, i would convert them.